api/tests/unit/app_analytics/split_testing/test_unit_split_testing_helpers.py

```diff
@@ -0,0 +1,31 @@
+import numpy as np
+from app_analytics.split_testing.helpers import analyse_split_test
+
+
+def test_analyse_split_tests_with_clear_results() -> None:
+    # Given
+    # Three options, with the last as clear winner as 400 / 500.
+    conversion_counts = [100, 200, 400]
+    evaluation_counts = [500, 500, 500]
+    input_data = np.array([conversion_counts, evaluation_counts])
```

Comment:
I might be missing something but it feels to me as though we could abstract any interactions with numpy from the user of these helper functions and just pass in the conversion counts and evaluation counts directly.

Reply:
Sure that's easy.